### PR TITLE
[Synapse]Reimport notebook Failure

### PR DESCRIPTION
**Pull request by zesluo**
**Description**

Import notebook to a new workspace Failure due to redundant parameters
**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Synapse

**Review by idear1203 — APPROVED** [review]

**Comment by zesluo** [comment]
@jsntcy  would you mind review this? Thanks in advance~^_^

File: src/azure-cli/azure/cli/command_modules/synapse/_help.py [hunk 1]
```diff
@@ -1438,7 +1438,7 @@
   - name: Create a notebook.
     text: |-
         az synapse notebook create --workspace-name testsynapseworkspace \\
-          --name testnotebook --file @path/notebook.json
+          --name testnotebook --file @path/notebook.ipynb
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/synapse/_help.py:1441]: Does that mean `.json` is preferred for notbook file? Can `.ipynb` file still work well?
I remember @idear1203 insisting on `.ipynb`. (See https://github.com/Azure/azure-cli/pull/17661#issuecomment-818489392)

Comment (idear1203) [reply src/azure-cli/azure/cli/command_modules/synapse/_help.py:1441]: @evelyn-ys , thanks for confirmation. Yes, we still want to use `.ipynb` in this command parameter. The reason of change here is, we want to correct the documentation to make it clear to our customers that `.ipynb` is the desired file format.

Comment (zesluo) [reply src/azure-cli/azure/cli/command_modules/synapse/_help.py:1441]: Pinging @evelyn-ys , thanks a lot!

**Pull request merged**